pages/linux/qm-list.md

```diff
@@ -0,0 +1,12 @@
+# qm list
+
+> List all virtual machines.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- List all virtual machines:
+
+`qm list`
+
+- List full status of active virtual machines:
+
+`qm list --full {{1}}`
```

Comment:
What `1` stands for here? If it's an `--full` value it's okay to write smth like this. But if it's a separate argument then it's better to write it's description in doubled curly braces I guess.

Reply:
```suggestion
`qm list --full 1`
```
It is a boolean value; So 1 is necessary for it to display the list. The argument braces aren't necessary here also other pages don't have them so I will revert my suggestion I made initially in this PR to use argument.

Ref

```text
qm list [OPTIONS]

Virtual machine index (per node).

--full <boolean>
Determine the full status of active VMs.
```

Reply:
Here's a deal: should we always provide `{{...}}` for any option values or not? To put it simple we can always emphasize that another option argument can be other than provided in our example via curly braces.

Reply:
Thanks for an answer. This should be mentioned in our style guide. Otherwise I or anybody else can repeat my mistake.